MODELS

```diff
@@ -55,6 +55,10 @@ Model     MCU         Name
 0422      attiny85    fireflies-pl47-219
 0423      attiny85    fireflies-pl47g2
 0441      attiny85    fireflies-e01
+0445      attiny85    fireflies-e12c
+0451      attiny1616  fireflies-lume1-6af       (E07X, E12R, NOV-Mu, etc)
+0452      attiny1616  fireflies-lume1-6af-nofet (T9R, T1R, etc)
```

Comment:
My SBT90A T9R which I just received from FFL, blinks 0451 as model, not 0452 as you indicated.

Is there a possible divergence in model numbers between what you're using here, and FFL's actual practice?

OTOH I'm pretty sure my T9R has FET as it blinks when ramping up, just before reaching maximum brightness -- so perhaps your model number is correct and consistent with FFL after all.

Reply:
Yes my friend, T9R with SBT90 has FET

Reply:
@DurvalMenezes, Light-Veteran is correct. The T9R comes in different emitter configurations. Some of them can take a much higher forward current such as your SBT-90, and therefore use the with-FET firmware. Others like the SFT-40 cannot and need to use the no-FET firmware. I had written a readme file in the Fireflies HW folder; hopefully that will help:
https://github.com/loneoceans/anduril/blob/trunk/hw/fireflies/README.md

Reply:
Thanks for the clarifications, @Light-Veteran and @loneoceans. 

Indeed, I've since compiled the fireflies-lume1-6af HEX and  flashed my SBT90.2 T9R with it, and everything seems good. Consulting the README.md was useful as I was able to match the revision number silk-screened in my T9R driver PCB ("REV C 10-22") with the exact model number it blinks during 15C, that is, 0451.

That said, I think the above-mentioned MODELS and README.md files could be clearer.  I will try and later submit a patch to that effect.

Reply:
> That said, I think the above-mentioned MODELS and README.md files could be clearer. I will try and later submit a patch to that effect.

@loneoceans I'm working on that. I think that just stating "a single small emitter" as in the currend README.md file is not the best; I think the best would be to indicate an objective number, like maximum forward current, which users can then consult on the datasheet for their particular LED, and decide on whether to flash the 'plain' or the nofet version.

Is it possible to indicate such a number?